schemas/plugins/AssetModulesPluginGenerator.json

```diff
@@ -34,6 +34,10 @@
           "$ref": "#/definitions/DataUrlFunction"
         }
       ]
+    },
+    "filename": {
+      "description": "Override for output.assetModuleFilename",
+      "type": "string"
```

Comment:
`assetModuleFilename` also allows functions. This is already supported by the current code, only the schema need to be extended.